presenters/keyword_test.go

```diff
@@ -0,0 +1,63 @@
+package presenters_test
+
+import (
+	"encoding/json"
+
+	_ "google-scraper/initializers"
+	"google-scraper/presenters"
+	. "google-scraper/tests"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Keyword", func() {
+	Describe("#InitializeKeywordPresenter", func() {
+		Context("given the valid keyword", func() {
+			It("returns the presenter object", func() {
+				user := FabricateUser("John", "john@example.com", "secret")
```

Comment:
@junan You can try with this [faker](https://github.com/bxcodec/faker) later when you want to improve it.
It really nice to use ✌️

Reply:
I will this improvement in another ticker and update all others tests ✌️

Reply:
OK fine 🧘‍♂️